### PR TITLE
update README.rst

### LINKED ISSUE #22: Add badge & link in README file for Riot chat room
It would be helpful for the README file to contain a badge link to the Matrix chat room, along with some text saying what the Matrix room is.  The current link is: https://riot.im/app/#/room/#pyastro_sunkit-dem:openastronomy.org  

I'd also suggest changing the primary link to be `#sunkit-dem:openastronomy.org` rather than `#pyastro_sunkit-dem:openastronomy.org`, since this will go beyond the PyAstro meeting.

### PR DESCRIPTION
**Pull request by sashank27**
Fixes #22

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,48 +1,49 @@
...
+License
+=======
+
+This project is Copyright (c) SunPy Developers and licensed under
+the terms of the BSD 3-Clause license. This package is based upon
+the `Openastronomy packaging guide <https://github.com/OpenAstronomy/packaging-guide>`_
+which is licensed under the BSD 3-clause licence. See the licenses folder for
+more information.
 
-**Imposter syndrome disclaimer**: We want your help. No, really.
```
Comment (PaulJWright) [inline README.rst:24]: I think we should keep this.

File: README.rst [hunk 2]
```diff
@@ -1,48 +1,49 @@
...
-.. image:: http://img.shields.io/badge/powered%20by-SunPy-orange.svg?style=flat
+|SunPy Badge| |matrix|
+
+.. |SunPy Badge| image:: http://img.shields.io/badge/powered%20by-SunPy-orange.svg?style=flat
     :target: http://www.sunpy.org
     :alt: Powered by SunPy Badge
+.. |matrix| image:: https://img.shields.io/matrix/sunpy:openastronomy.org.svg?colorB=%23FE7900&label=Chat&logo=matrix&server_fqdn=openastronomy.modular.im
+   :target: https://riot.im/app/#/room/#sunkit-dem:openastronomy.org
 
-License
```
Comment (PaulJWright) [inline README.rst:8]: probably shouldn't be removing the license?

Comment (wtbarnes) [reply README.rst:8]: I think it's just been moved below

File: README.rst (hunk 1 above)
Comment (wtbarnes) [reply README.rst:24]: Agreed. Let's not delete this

File: README.rst (hunk 2 above)
Comment (PaulJWright) [reply README.rst:8]: Ah yes, apologies.

**Pull request merged**